exporter/kafkaexporter/kafka_exporter.go

```diff
@@ -128,8 +128,7 @@ func newSaramaProducer(config Config) (sarama.SyncProducer, error) {
 	// These setting are required by the sarama.SyncProducer implementation.
 	c.Producer.Return.Successes = true
 	c.Producer.Return.Errors = true
-	// Wait only the local commit to succeed before responding.
-	c.Producer.RequiredAcks = sarama.WaitForLocal
+	c.Producer.RequiredAcks = config.Producer.RequiredAcks
```

Comment:
Is this the default behavior of the underlying lib? In any case, you need an entry in the changelog about this change.

Reply:
Yes this is the default. https://github.com/Shopify/sarama/blob/main/config.go#L167